app/models/log_file.rb

```diff
@@ -140,19 +140,12 @@ def self.empty_logfile(historical)
                    :description => "Default logfile")
   end
 
-  # Added tcp ping stuff here until ftp is refactored into a separate class
-  def self.get_ping_depot_options
-    @@ping_depot_options ||= VMDB::Config.new("vmdb").config[:log][:collection]
-  end
-
   def self.ping_timeout
-    get_ping_depot_options
-    @@ping_timeout ||= (@@ping_depot_options[:ping_depot_timeout] || 20)
+    @ping_timeout = ::Settings.log.collection.ping_depot_timeout
   end
 
   def self.do_ping?
-    get_ping_depot_options
-    @@do_ping ||= @@ping_depot_options[:ping_depot] == true
+    @do_ping = ::Settings.log.collection.ping_depot == true
```

Comment:
You can remove the `@do_ping` ivar as it's not used elsewhere, and there is not longer a need for memoization.